http4k-core/src/main/kotlin/org/http4k/filter/ClientFilters.kt

```diff
@@ -182,7 +182,7 @@ object ClientFilters {
             val redirect = ensureValidMethodForRedirect().uri(newUri)
 
             return when {
-                header("host") != null && newUri.host.isNotEmpty() -> redirect.replaceHeader("host", newUri.hostAndPort)
+                header("host") != null && newUri.host.isNotEmpty() -> redirect.replaceHeader("host", newUri.host)
```

Comment:
What about this one? Here a `host` header is set, so shouldn't it include the `port` as well?

Reply:
Missed that one. Will clean it up for the next release 🙃